theia-deb-build-docker/package.json

```diff
@@ -4,7 +4,7 @@
     "license": "EPL-2.0",
     "description": "A custom Theia debian package",
     "engines": {
-        "node": "10.x"
+        "node": "12.x"
```

Comment:
I'm not sure if we should try to enforce a higher version of `node` like other images and the upstream repository:

https://github.com/theia-ide/theia-apps/blob/0610ffa4213c499127b372d92dc658ec726d9ca1/theia-full-docker/Dockerfile#L5

- https://github.com/eclipse-theia/theia/blob/d145b055edc304bf3c9a18cc6b57be66018fcf48/package.json#L7

It may not be an issue as other images use `12` as well:

https://github.com/theia-ide/theia-apps/blob/0610ffa4213c499127b372d92dc658ec726d9ca1/theia-swift-docker/Dockerfile#L1